sqlglot/dialects/clickhouse.py

```diff
@@ -328,3 +352,27 @@ def parameterizedagg_sql(self, expression: exp.Anonymous) -> str:
 
         def placeholder_sql(self, expression: exp.Placeholder) -> str:
             return f"{{{expression.name}: {self.sql(expression, 'kind')}}}"
+
+        def oncluster_sql(self, expression: exp.OnCluster) -> str:
+            return f"ON CLUSTER {self.sql(expression, 'this')}"
+
+        def createable_sql(
+            self,
+            expression: exp.Create,
+            locations: dict[exp.Properties.Location, list[exp.Property]],
+        ) -> str:
+            kind = self.sql(expression, "kind").upper()
+            if kind in self.ON_CLUSTER_TARGETS and locations.get(exp.Properties.Location.POST_NAME):
+                this_name = self.sql(expression.this, "this")
+                this_properties = self.properties(
+                    exp.Properties(expressions=locations[exp.Properties.Location.POST_NAME]),
+                    wrapped=False,
+                )
+                schema_exp = self.expressions(expression.this)
+                if self.pretty:
+                    this_schema = f"{self.sep()}({self.seg(schema_exp)}{self.sep()})"
+                else:
+                    this_schema = f"({schema_exp})"
+                return f"{this_name} {this_properties} {this_schema}"
+            else:
```

Comment:
else is not needed

Reply:
Just `return super()` ? Seemed less clear to me, but ok

Reply:
it's a pylint standard to do

if ...:
   return 
return ...

Reply:
yeah, that's how it was initially. :)